tests/pytorch/fused_attn/test_fused_attn.py

```diff
@@ -191,6 +191,12 @@ def test():
     "base_2_1": ModelConfig(1, 24, 24, 128, 2048, 4096, 0.0, "no_mask", "no_bias"),  # cross, 1
     "base_3_0": ModelConfig(8, 16, 16, 128, 1, 2048, 0.0, "no_mask", "no_bias"),  # inference
     "base_3_1": ModelConfig(8, 16, 16, 256, 1, 2048, 0.0, "no_mask", "no_bias"),  # inference
+    "base_11_0": ModelConfig(8, 16, 16, 64, 128, 128, 0.0, "causal", "no_bias"),  # self , 0
+    "base_11_1": ModelConfig(4, 16, 16, 64, 128, 256, 0.0, "causal", "no_bias"),  # cross, 0
+    "base_12_0": ModelConfig(2, 24, 24, 128, 2048, 2048, 0.0, "causal", "no_bias"),  # self , 1
+    "base_12_1": ModelConfig(1, 24, 24, 128, 2048, 4096, 0.0, "causal", "no_bias"),  # cross, 1
+    "base_13_0": ModelConfig(8, 16, 16, 128, 1, 2048, 0.0, "causal", "no_bias"),  # inference
+    "base_13_1": ModelConfig(8, 16, 16, 256, 1, 2048, 0.0, "causal", "no_bias"),  # inference
```

Comment:
Not sure I understand the naming for these?

Reply:
It's removed, and some inference tests are added to `test_dpa_mask`.